### PR TITLE
Add Irshad Ali

### PR DESCRIPTION
**Pull request by phonerefer**
Hello 👋 I'm Irshad Ali
👨‍💻 Front-End-Developer based in Hyderabad, India. With a passion for creating new ideas and beautiful functional websites with a strong understanding of development.

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -1,5 +1,25 @@
 // keep it commonjs export
 module.exports = [
+  {
+    name: 'Irshad Ali',
+    description: 'Front-End-Developer based in Hyderabad, India.',
+    url: 'https://www.irshadali.site/Uses',
```
Comment (sam-cross) [inline src/data.js]: Please make `Uses` lowercase - thanks!

File: src/data.js [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+    emoji: '👋',
+    country: '🇮🇳',
+    computer: 'windows',
+    phone: 'Android',
+    tags: [
+      'Developer',
+      'Designer',
+      'Blogger',
+	  'JavaScript',
+	  'SvelteJS'
```
Comment (sam-cross) [inline src/data.js]: Missing comma after `SvelteJS`.

Comment (sam-cross) [reply src/data.js]: Please also change this to `'Svelte'` to keep entries consistent.

File: src/data.js [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+    twitter: '@imirshadali',
+    emoji: '👋',
+    country: '🇮🇳',
+    computer: 'windows',
+    phone: 'Android',
+    tags: [
+      'Developer',
+      'Designer',
+      'Blogger',
+	  'JavaScript',
```
Comment (sam-cross) [inline src/data.js]: Please keep indentation consistent

**Comment by yevhenorlov** [comment]
resolved in https://github.com/wesbos/awesome-uses/pull/691

**Pull request closed without merging**